.github/workflows/solidity-foundry.yml

```diff
@@ -23,6 +23,7 @@ jobs:
               - 'contracts/src/v0.8/**/*'
               - '.github/workflows/solidity-foundry.yml'
               - 'contracts/foundry.toml'
+              - 'contracts/gas-snapshots/*.gas-snapshot'
```

Comment:
This was needed, otherwise CI wasn't running the foundry tests upon a snapshot change.